### PR TITLE
Extend signature event types with more data

### PR DESCRIPTION
**Pull request by patrickpichler**
In order to better understand what is happening, the signature events reported to the backend now hold more info, such as fd of the tty opened, or the ip used for the stdio over socket detection.

### CODE REVIEW

File: pkg/ebpftracer/signature/tty_detected_test.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+					Sockfd: 10,
+					Type:   0,
+					RemoteAddr: types.Ip4SockAddr{
+						Addr: netip.MustParseAddrPort("1.2.3.4:1190"),
+					},
+				},
+			},
+		},
+		{
+			title: "should fire for socket dup event with new fs being stdio",
```
Comment (anjmao) [inline pkg/ebpftracer/signature/tty_detected_test.go:75]: This test looks like not related to tty

**Review by anjmao — APPROVED** [review]

File: pkg/ebpftracer/signature/tty_detected_test.go (hunk 1 above)
Comment (patrickpichler) [reply pkg/ebpftracer/signature/tty_detected_test.go]: Good catch! Test case has been removed, haha.
